File: snippets/node/embed-jobs-cancel.ts
const { CohereClient } = require('cohere-ai');

const cohere = new CohereClient({ token: process.env.COHERE_API_KEY! });

(async () => {
  const embedJob = await cohere.embedJobs.cancel('job_id');

  console.log(embedJob);
})();

File: snippets/java/app/src/main/java/EmbedJobsCancel.java
import com.cohere.api.Cohere;


public class EmbedJobsCancel {
    public static void main(String[] args) {
        Cohere cohere = Cohere.builder().token(System.getenv("CO_API_KEY")).clientName("snippet").build();

        cohere.embedJobs().cancel("job_id");
    }
}
